source/views/news/index.tsx

```diff
@@ -27,14 +27,17 @@ const OlevilleNewsView = () => (
 
 const NewsView = (): JSX.Element => {
 	return (
-		<Tabs.Navigator>
+		<Tabs.Navigator
+			screenOptions={{
+				headerShown: false,
+			}}
+		>
```

Comment:
```suggestion
		<Tabs.Navigator screenOptions={{ headerShown: false }}>
```

or does Prettier complain about the online version?

Or did you try it and dislike it?

Reply:
I like your spacing suggestion a lot better. Prettier likes it, too. Let's go with that!